charts/kubewarden-controller/values.yaml

```diff
@@ -73,10 +73,11 @@ preDeleteHook:
     runAsNonRoot: true
 # open-telemetry options
 telemetry:
-  enabled: False
   metrics:
+    enabled: False
     port: 8080
   tracing:
+    enabled: False
```

Comment:
nitpicking: I see we use `true` above, shouldn't we use `false` to stay consistent with the naming convention?

Reply:
Good point, done in https://github.com/kubewarden/helm-charts/pull/331/commits/50e00feecfbec124d0e2ad840534745aa1ce5ff3.